packages/react/src/components/ComposedModal/ComposedModal.js

```diff
@@ -369,29 +369,32 @@ export class ModalHeader extends Component {
   }
 }
 
-export class ModalBody extends Component {
-  static propTypes = {
-    /**
-     * Specify an optional className to be added to the Modal Body node
-     */
-    className: PropTypes.string,
-  };
-
-  render() {
-    const { className, children, ...other } = this.props;
-
-    const contentClass = classNames({
-      [`${prefix}--modal-content`]: true,
-      [className]: className,
-    });
-
-    return (
-      <div className={contentClass} {...other}>
-        {children}
-      </div>
-    );
-  }
+export function ModalBody(props) {
+  const { className, children, hasScrollingContent, ...other } = props;
+  const contentClass = classNames({
+    [`${prefix}--modal-content`]: true,
+    [className]: className,
+  });
+  return (
+    <div
+      className={contentClass}
+      tabIndex={hasScrollingContent ? 0 : null} // eslint-disable-line jsx-a11y/no-noninteractive-tabindex
```

Comment:
When the `tabIndex="0"`, I believe that `role="region"` and an aria label need to be added too

Reply:
https://developer.paciellogroup.com/blog/2016/02/short-note-on-improving-usability-of-scrollable-regions/